scss/base/_common_classes.scss

```diff
@@ -0,0 +1,45 @@
+* {
+  box-sizing: border-box;
+}
+
+.site__content-container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  @include text-formatter;
+}
+
+.block__container {
+  display: flex;
+  width: 100%;
+  max-width: 1126px;
+}
+
+.primary-title {
```

Comment:
Well done!
You can separate layout classes and typography classes in corresponding files.